### PR TITLE
Update VERIFICATION.md

### PR DESCRIPTION
**Pull request by anvacaru**
closes: #2289

### CODE REVIEW

**Comment by anvacaru** [comment]
Most of the Solidity examples in `test/specs/examples` require the `solc-to-k` command that was extracted in kontrol. We should either remove them or bring back some of the `solc-to-k` functionality.

File: VERIFICATION.md [hunk 1]
```diff
@@ -18,95 +20,52 @@ It has two modules:
...
-
-In addition to this, you can use `kevm show-kcfg ...` or `kevm view-kcfg ...` to get a visualization.
+For `kevm prove`. you can use `kevm show-kcfg ...` or `kevm view-kcfg ...` to get a visualization.
+***Note:*** this is not compatible with `kevm prove-legacy`.
 
 `kevm view-kcfg [spec_file] [--save-directory save_directory] [--claim claim_label] ...` command takes the same basic arguments as `kevm prove ...` does, including:
-  - `spec_file` is the file to look in for specifications. This file is read like with `kevm prove —pyk …`; the `KProve.get_claims` invokes the frontend.
-  - `--save_directory` must be passed as where the KCFGs have been saved (by a previous call to `kevm prove --save-directory save_directory ...`
-  - `--claim claim_label` option is added, but unlike the `kevm prove ...`, you can only repeat it once. This option lets you select an individual claim out of the `spec_file`; if not supplied, it’s assumed that only one spec is present.
+  - `spec_file` is the file to look in for specifications. This is the same file used for `kevm prove …`.
```
Comment (palinatolmach) [inline VERIFICATION.md:50]:
```suggestion
  - `spec_file` is the file to look in for specifications. This is the same file that is used for `kevm prove …`.
```

File: VERIFICATION.md [hunk 2]
```diff
@@ -18,95 +20,52 @@ It has two modules:
...
-```
-
-You can also debug proofs interactively:
-
-```sh
-kevm prove tests/specs/erc20/ds/transfer-failure-1-a-spec.k --definition tests/specs/erc20/verification/haskell --debugger
-```
-
-In addition to this, you can use `kevm show-kcfg ...` or `kevm view-kcfg ...` to get a visualization.
+For `kevm prove`. you can use `kevm show-kcfg ...` or `kevm view-kcfg ...` to get a visualization.
```
Comment (palinatolmach) [inline VERIFICATION.md:46]:
```suggestion
For `kevm prove`, you can use `kevm show-kcfg ...` or `kevm view-kcfg ...` to get a visualization.
```

File: VERIFICATION.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
 Verification Instructions for KEVM
 ==================================
 
+If you're trying to run Solidity programs with symbolic execution, check out our guide for KONTROL at [docs.runtimeverification.com/kontrol].
```
Comment (palinatolmach) [inline VERIFICATION.md:4]:
```suggestion
If you're trying to analyze Solidity smart contracts with symbolic execution, check out our guide for Kontrol at [docs.runtimeverification.com/kontrol].
```

**Review by palinatolmach — APPROVED** [review]
Approved with some wording suggestions (please feel free to ignore!)

**Pull request merged**